configure.ac

```diff
@@ -40,6 +40,11 @@ AC_C_BIGENDIAN
 # enable the automatically build of shared and static libraries
 LT_INIT([shared static])
 
+#check for struct cmsghdr
```

Comment:
Is there a reason why the check has been inserted here? Usually, the type checks are in the configure option independent section after the header checks (cf. line 572 ff).

Reply:
My mistake - I looked for AC_CHECK_TYPES, found none and so put it higher up.  Now corrected.